Apps/WebClient/src/ClientApp/src/components/laboratory/covid19LaboratoryTestDescription.vue

```diff
@@ -26,10 +30,16 @@ export default class Covid19LaboratoryTestDescriptionComponent extends Vue {
         >
             <span>{{ paragraph }}</span>
             <span v-if="shouldDisplayLink(index)">
-                <a :href="link" data-testid="result-link" target="blank_"
-                    >this page</a
-                >.
+                <a data-testid="result-link" @click="displayLink">this page</a>.
             </span>
         </p>
     </div>
 </template>
+
+<style scoped>
+a {
+    color: blue !important;
+    text-decoration: underline !important;
+    cursor: pointer !important;
+}
+</style>
```

Comment:
We use the $hg-link colour for our text links, but rather than changing the colour we set, I think we can get normal link colouring and underlining if we give the `<a>` element a class tag.  We would still need to change the cursor, though.  (Applies to each page.)